examples/cryptocurrency-advanced/docker/launch.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+node_count=4
+start_peer_port=6331
+start_public_port=8200
+start_private_port=8091
+
+cd backend && mkdir example && cd example
+/root/.cargo/bin/exonum-cryptocurrency-advanced generate-template common.toml --validators-count 4
```

Comment:
I suppose a good way is to create a variable that stores path `/root/.cargo/bin/exonum-cryptocurrency-advanced` and use it in the script.